tests/e2e/nvidia-container-cli_test.go

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"text/template"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	libnvidiaContainerCliDockerRunTemplate = `
+docker run -d --name test-nvidia-container-cli \
+  --privileged \
+  --runtime=nvidia \
```

Comment:
It is not scalable to have to MOUNT everything into this container. Note that when we still had some simple integration tests in the toolkit we used https://github.com/NVIDIA/nvidia-container-toolkit/blob/27f5ec83debaa8a173e90c43b013d5acad4b27a0/tests/container/docker_test.sh#L20

Can we rather adapt this?

Reply:
sure, will work on that. we need to make this test more robust and scalable